lib/axp209.js

```diff
@@ -85,11 +96,38 @@ AXP209.prototype._readButton = function() {
   return (value ? 1 : 0);
 };
 
+AXP209.prototype._readVcc18Adc = function() {
+  // enable 12-bit ADC for VCC 1.8 
+  var p = this._i2c.readRegister(0x85, 1)[0]; // 0-2v range or .7 - 2.7v range?
+  var rhigh = this._i2c.readRegister(0x64, 1)[0].toString(16);
+  var rlow = this._i2c.readRegister(0x65, 1)[0].toString(16);
+  var r = parseInt(rhigh + rlow, 16);
+  
+  var voltage = (((r * 10000)/4096) * 2) + (p * 7000); // voltage in millivolts
+  
+  return voltage;
```

Comment:
Just wondering if we should be return a 12-bit number here instead of the voltage.